src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -39,6 +39,7 @@ static void EnsureLibSsl10Initialized()
 #endif
 
 static int32_t g_config_specified_ciphersuites = 0;
+static char* emptyAlpn = "";
```

Comment:
I believe the convention for C globals is g_

```suggestion
static char* g_emptyAlpn = "";
```